src/config-file-watcher.ts

```diff
@@ -21,9 +21,7 @@ class ConfigFileWatcherFeature implements vscodelc.StaticFeature {
 
   initialize(capabilities: vscodelc.ServerCapabilities,
              _documentSelector: vscodelc.DocumentSelector|undefined) {
-    if ((capabilities as ClangdClientCapabilities)
-            .compilationDatabase?.automaticReload)
```

Comment:
I'd like us to keep respecting the server's `automaticReload` capability by default.

I was thinking of adding a new config option, like `clangd.onConfigChanged.forceEnable: true`, and only skipping this `automaticReload` check if that option is enabled.